library/default-synths-extra.scd

```diff
@@ -294,3 +294,120 @@
 	}).add
 );
 
+
+// tuning fork
+// from my own experimentation and from "On the acoustics of tuning forks", Rossing Russell and Brown
+(SynthDef(\superfork, {|out, sustain=1, pan, freq, accelerate=0|
```

Comment:
you may want to add a newline after the inital parens, more convenient for those who select by double click.